src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -265,18 +287,101 @@ const NetworkModificationNodeEditor = ({ selectedNode }) => {
 
     useEffect(() => {
         if (selectedNode !== selectedNodeRef.current) {
+            // loader when opening a modification panel (current user only)
+            setLaunchLoader(true);
             selectedNodeRef.current = selectedNode;
-            if (!selectedNode.networkModification) setModifications([]);
-            else {
+            if (!selectedNode.networkModification) {
+                setModifications([]);
+                setLaunchLoader(false);
+            } else {
                 fetchNetworkModifications(selectedNode.networkModification)
                     .then((res) => {
                         if (selectedNodeRef.current === selectedNode)
                             setModifications(res.status ? [] : res);
                     })
-                    .catch((err) => snackError(err.message));
+                    .catch((err) => snackError(err.message))
+                    .finally(() => {
+                        setLaunchLoader(false);
+                    });
             }
         }
-    }, [selectedNode, setModifications, selectedNodeRef, snackError]);
+    }, [
+        selectedNode,
+        setModifications,
+        selectedNodeRef,
+        snackError,
+        dispatch,
+        errorList,
```

Comment:
why do you have a dep to errorList here ?

Reply:
my bad, ill remove it